src/lib/tokenUtils.ts

```diff
@@ -1,43 +1,19 @@
 import { ethers } from "ethers";
 import { OptimismContracts } from "./contracts/contracts";
-import { DEPLOYMENT_NAME, Deployments } from "./config";
+import Tenant from "@/lib/tenant";
 
 // TODO: This file seems messy -- consider refactoring
 
-export const tokens: Map<
-  Deployments | string,
-  { name: string; symbol: string; decimals: number }
-> = new Map([
-  [
-    "optimism",
-    {
-      name: "Optimism",
-      symbol: "OP",
-      decimals: 18,
-    },
-  ],
-  [
-    "0x4200000000000000000000000000000000000042",
```

Comment:
We still might need to resolve tokens by contract address when resolving proposals transaction data